### PR TITLE
Add transfer-sh_upload extension

### PR DESCRIPTION
**Pull request by 0PandaDEV**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/transfer-sh_upload/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    // After upload completes, handle the server's response or perform additional actions as needed
+    // Clipboard.copy(responseUrl) // Adjust this line based on your actual response handling
+    const response = await uploadStream;
+    await Clipboard.copy(response.body);
+
+    await showToast({
+      style: Toast.Style.Success,
+      title: "Upload successful",
+      message: fileName,
```
Comment (pernielsentikaer) [inline extensions/transfer-sh_upload/src/index.ts:79]: Can we show the `Copied URL to clipboard` instead?

**Review by pernielsentikaer — COMMENTED** [review]
Can we handle the crash if you enters something that's not an URL?

Raycast07022024-WG7xwcbX png@2x

File: extensions/transfer-sh_upload/src/index.ts (hunk 1 above)
Comment (0PandaDEV) [reply extensions/transfer-sh_upload/src/index.ts:79]: Instead of "Upload Successful" ?

Comment (pernielsentikaer) [reply extensions/transfer-sh_upload/src/index.ts:86]: Yes 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**